chart/templates/mayastor/agents/ha/ha-node-daemonset.yaml

```diff
@@ -57,7 +61,7 @@ spec:
         args:
         - "--node-name=$(MY_NODE_NAME)"
         - "--grpc-endpoint=$(MY_POD_IP):11600"
-        - "--cluster-agent=https://mayastor-agent-ha-cluster:11500"
+        - "--cluster-agent=https://mayastor-agent-core:11500"
```

Comment:
Should this be {{ .RELEASE.NAME }}-agent-core instead of "mayastor-agent-core" ?

Reply:
Right. It should be. Will change it. Thank you.

Reply:
Done.